python/foglamp/services/south/server.py

```diff
@@ -102,7 +102,7 @@ class Server:
                 raise
 
             try:
-                cls._plugin = __import__("foglamp.device.{}_device".format(plugin_module_name), fromlist=[''])
+                cls._plugin = __import__("foglamp.plugins.south.{}.{}".format(plugin_module_name,plugin_module_name), fromlist=[''])
```

Comment:
@MarkRiddoch there isn't any new documentation on how this should be so I've kept it this way. Please advice.